CHANGES.rst

```diff
@@ -95,6 +95,9 @@ There's a frood who really knows where his towel is.
 - Changes image link behavior in view.pt for mobile devices
   (closes `#62`_). [marcosfromero]
 
+- Replace Galleria with Cycle2 (closes `#116`_).
```

Comment:
changelog lines on top of the others: http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#tracking-changes